tests/tensor_search/test_multimodal_tensor_combination.py

```diff
@@ -62,13 +62,6 @@ def test_add_documents(self):
         tensor_search.add_documents(config=self.config, add_docs_params=AddDocsParams(
             index_name=self.index_name_1, docs=[
                 expected_doc,
-                # this is just a dummy one
-                {
-                    "Title": "Horse rider",
-                    "text_field": "A rider is riding a horse jumping over the barrier.",
-                    "image_field": "https://marqo-assets.s3.amazonaws.com/tests/images/image1.jpg",
-                    "_id": "1"
-                },
```

Comment:
why was this removed?

Reply:
it doesn't actually add anything to the test. Just seems like an extraneous doc. But I can put it back, it may just be to check if the correct document was fetched.